cli_tutorials/cli_charge_molecules.md

```diff
@@ -0,0 +1,105 @@
+# Generating Partial Charges with the OpenFE CLI
+
+It is recommended to use a single set of charges for each ligand to ensure reproducibility between repeats or consistent 
+charges between different legs of a calculation involving the same ligand, like a relative binding affinity calculation for example. 
+As such both the `plan-rbfe-network` and `plan-rhfe-network` commands will calculate partial charges for ligands making it expensive 
+to run multiple network mappings while finding the optimal one for the resources available. 
+
+Here we present a CLI tool to allow you to do this ahead of time, reducing overheads and further improving reproducibility.
```

Comment:
"do this" -> calculate partial charges

There's nothing wrong with this sentence, here my suggestion is purely based on the assumption that most readers don't tend to have enough time to link the context of multiple paragraphs together.